srv/components/ha/haproxy/config.sls

```diff
@@ -58,5 +58,10 @@ Setup haproxy config to enable logs:
     - makedirs: True
     - keep_source: False
 
+Stop haproxy service:
```

Comment:
No
Just add
`- components.ha.haproxy.stop ` in init.sls

Reply:
Done